examples/Makefile-nrf52840

```diff
@@ -79,6 +79,10 @@ COMMONCFLAGS                   := \
     -I$(CONFIG_FILE_PATH)         \
     $(NULL)
 
+ifeq ($(CERT_LOG),1)
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT=1
```

Comment:
I see. Makes sense. I agree the current way seems ok (definitely easier and simpler). 
Thanks @jwhui.